### PR TITLE
OEP 0018: reminder that pip install from PyPI ignores URLs

### PR DESCRIPTION
**Pull request by singuliere**

### CODE REVIEW

**Comment by edx-webhook** [comment]
Thanks for the pull request, @singuliere! I've created [OSPR-3073](https://openedx.atlassian.net/browse/OSPR-3073) to keep track of it in JIRA. JIRA is a place for product owners to prioritize feature reviews by the engineering development teams.

Feel free to add as much of the following information to the ticket:
- supporting documentation
- edx-code email threads
- timeline information ("this must be merged by XX date", and why that is)
- partner information ("this is a course on edx.org")
- any other information that can help Product understand the context for the PR

All technical communication about the code itself will still be done via the GitHub pull request interface. As a reminder, [our process documentation is here](http://edx-developer-guide.readthedocs.org/en/latest/process/overview.html).

**Comment by natabene** [comment]
@singuliere Thank you for your contribution. @nasthagiri Can you give this a look? CCing @ormsbee  just in case

**Comment by nasthagiri** [comment]
@jmbowman @cpennington Can either of you take a look at this PR?  Thanks!

File: oeps/oep-0018-bp-python-dependencies.rst [hunk 1]
```diff
@@ -400,6 +400,8 @@ packages from URLs whenever possible:
   partially worked around via a post-processing script for the generated
   requirements files; an example can be found in `edx-platform`_ at
   ``scripts/post-pip-compile.sh``.
+* When installing a package from PyPI, pip will not pull requirements
+  from URLs. It will only pull requirements from PyPI.
```
Comment (jmbowman) [inline oeps/oep-0018-bp-python-dependencies.rst]: Thanks, I'd missed that point.  The [full story](https://github.com/pypa/pip/issues/4187) behind this is pretty long and winding, but maybe this note should just mention that this is for security reasons (not allowing PyPI packages to fetch code from URLs with mutable content).

Comment (singuliere) [reply oeps/oep-0018-bp-python-dependencies.rst]: Added the part about mutable contents.

**Comment by jmbowman** [comment]
Could you update the "Last Modified" date near the top of the file?  Should be good to merge after that.

**Comment by singuliere** [comment]
@jmbowman indeed, I overlooked this. Updated & repushed!

**Comment by edx-webhook** [comment]
@singuliere 🎉 Your pull request was merged!

Please take a moment to [answer a two question survey](https://docs.google.com/forms/d/e/1FAIpQLSceJOyGJ6JOzfy6lyR3T7EW_71OWUnNQXp68Fymsk3MkNoSDg/viewform?usp=pp_url&entry.1671973413=edx/open-edx-proposals&entry.867055334=https://github.com/edx/open-edx-proposals/pull/106&entry.1484655318=https://github.com/singuliere&entry.752974735=2019-02-12+10:16&entry.1917517419=2019-02-20+13:38&entry.2133058324=Yes) so we can improve your experience in the future.

**Pull request merged**